src/handler/rest/client.ts

```
@@ -0,0 +1,56 @@
+                method: RouteMethod.Get,
+                path: ClientHandler.path,
+                func: this.list.bind(this),
+                needAuth: true
+            }
+        ]
+    }
+
+    public list(req: Request, res: Response): void{
+        const[err, body] = validate<DefaultQuery>(DefaultQuerySchema, req.query)
+        if (err) {
+            return errorResponse(res, err)
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)